client/app/views/contacts/components/data.coffee

```diff
@@ -0,0 +1,58 @@
+DatapointsView  = require 'views/contacts/components/datapoints'
+XtrasView       = require 'views/contacts/components/xtras'
+EditActionsView = require 'views/contacts/components/edit_actions'
+
+CONFIG = require('config').contact
+
+
+module.exports = class DataView extends Mn.LayoutView
+
+    template: require 'views/templates/contacts/components/data'
+
+
+    behaviors:
+        Form: {}
+
+
+    ui:
+        formAdd: '.actions button'
+
+
+    modelEvents:
+        'change': -> @render() unless @model.get 'edit'
+
+
+    initialize: ->
+        Mn.bindEntityEvents @model, @, @model.viewEvents
+
+
+    serializeData: ->
+        _.extend {}, super, fullname: @model.toString pre: '<b>', post: '</b>'
+
+
+    _renderDatapoints: ->
+        @$('.datapoints').each (index, el) =>
```

Comment:
Not sure what you are doing there, but aren't you pulling data from the DOM ? Could you instead have datapoints as an array somewhere and use the array to contruct the DOM ?